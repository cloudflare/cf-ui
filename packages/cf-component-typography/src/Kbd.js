const React = require('react');
const { PropTypes } = React;

class Kbd extends React.Component {
  render() {
    return (
      <kbd className="cf-kbd">
        {this.props.children}
      </kbd>
    );
  }
}

Kbd.propTypes = {
  children: PropTypes.node
};

module.exports = Kbd;
