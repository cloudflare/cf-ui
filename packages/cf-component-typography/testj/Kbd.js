import React from 'react';
import renderer from 'react-test-renderer';
import Kbd from '../src/Kbd';

describe('Kbd', () => {
  test('should render', () => {
    const component = renderer.create(<Kbd>Kbd</Kbd>);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
