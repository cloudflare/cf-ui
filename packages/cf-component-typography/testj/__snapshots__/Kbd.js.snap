// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Kbd should render 1`] = `
<kbd
  className="cf-kbd"
>
  Kbd
</kbd>
`;
